src/components/Picker/index.js

```diff
@@ -102,10 +104,28 @@ class Picker extends PureComponent {
         };
 
         this.onInputChange = this.onInputChange.bind(this);
+        this.placeholder = this.props.placeholder;
+        this.items = this.props.items;
     }
 
     componentDidMount() {
         this.setDefaultValue();
+
+        // Windows will reuse the text color of the select for each one of the options
```

Comment:
It's probably not very ideal to have this logic here and I think the constructor is a better place for it. The reason is,  `componentDidMount()` isn't run until after it's rendered, but we want those values to be there before things are rendered.